src/TableV8/TablePagination/index.tsx

```diff
@@ -75,10 +77,13 @@ export const TablePaginationV8 = ({
                 min-width: 70px;
               }
             `}
-            onChange={(pageSizeValue: string) => setPageSize(() => Number(pageSizeValue))}
+            onChange={(pageSizeValue: string) => {
+              setPageSize(() => parseInt(pageSizeValue));
```

Comment:
change page size back to number for state & API purposes

Reply:
why is pageSizeValue a string? looks like a lot of code that's parsing to an number and then changing to a string.
I think it should always be a number for logic and can be a string for display eg. pageSize.toString in a popper tooltip

Reply:
it's a string in the HTML element and a number in state so I have to change it back and forth

Reply:
> it's a string in the HTML element and a number in state so I have to change it back and forth

I don't understand what you mean by the HTML element. Sure every time it's a HTML element, display as a string. This works though right?: 
```javascript
<Select
            aria-label="Select page size"
            css={css`
              z-index: 10;
              transform: scale(0.8);
              & [role='button'] {
                min-width: 70px;
              }
            `}
            onChange={() => {
              setPageSize(pageSize);
              resetPageIndex();
            }}
            options={pageSizeOptions.map((option: number) => ({
              content: option,
              value: option,
            }))}
            popupPosition={POPUP_POSITIONS.UP}
            value={pageSize.toString()}
          />
```